### ISSUE TITLE
Allow CLI to unbundle archives into separate folders like old CLI

### ISSUE THREAD

**Issue by 86maylin** [WolvenKit/WolvenKit#1589]
**Is your feature request related to a problem? Please describe.**
I've made a custom script that packs mods in bulk when they are in their own folders which is helpful for mods with variants, and it works well with the old CLI I'm using, but I need to use newer ones for ep1 paths yet they unbundle multiple archives into a single folder and overrides files with same path. 

**Describe the solution you'd like**
I'd like to have the option to unbundle archives into separate folders, maybe with an extra parameter where if it's provided it'll do that.

**Comment by rfuzzo** [comment]
You can still target the archives individually with the CLI and write a bat file for it.

But I'm not sure I understand the request maybe: so you want to target a folder and then for each archive in it, unbundle it into a subfolder?

**Comment by 86maylin** [comment]
Yes! Say I have something.archive and someotherthing.archive in a folder, if I run unbundle on that folder, old CLI will give me something and someotherthing folder with their respective files in there, while new CLI will just give me a base folder with both files in merged there. 
I also feel like unbundling to separate folder can be useful to other people as an option.
